packages/web-app/src/lib/api-wrapper.js

```diff
@@ -2,7 +2,7 @@ const getData = (appealId) => {
   if (appealId) {
     return {
       appeal: {
-        id: '1dc8a211-4e1b-44fc-9e9c-a6e7b802fb8a',
+        id: appealId || '1dc8a211-4e1b-44fc-9e9c-a6e7b802fb8a',
```

Comment:
The addition of this condition for the scenario where `appealId` is not set and the default value is used is not covered in the tests, please add a test to cover this.